Mailer/TwigSwiftMailer.php

```diff
@@ -119,4 +119,27 @@ protected function sendMessage($templateName, $context, $fromEmail, $toEmail)
 
         $this->mailer->send($message);
     }
+
+    /**
+     * Send confirmation link to specified new user email.
+     * @param UserInterface $user
+     * @param $confirmationUrl
+     * @param $toEmail
+     * @return bool
+     */
+    public function sendUpdateEmailConfirmation(UserInterface $user, $confirmationUrl, $toEmail)
```

Comment:
This function should be defined in the MailerInterface and also implemented in the Mailer and NoopMailer classes.